contracts/InterestRate/StableRate.sol

```diff
@@ -0,0 +1,248 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity 0.8.13;
+
+/**
+ * @title Logic for Venus stable rate.
+ */
+contract StableRateModel {
+    event NewStableInterestParams(
+        uint256 baseRatePerBlockForStable,
+        uint256 multiplierPerBlock,
+        uint256 multiplierPerBlockForStable,
+        uint256 jumpMultiplierPerBlockForStable,
+        uint256 kink,
+        uint256 stableRatePremium,
+        uint256 optimalStableLoanRate
+    );
+
+    /// @notice Indicator that this is an InterestRateModel contract (for inspection)
+    bool public constant isInterestRateModel = true;
+
+    uint256 private constant BASE = 1e18;
+
+    /**
+     * @notice The approximate number of blocks per year that is assumed by the interest rate model
+     */
+    uint256 public constant blocksPerYear = 2102400;
+
+    /**
+     * @notice The stable base interest rate which is the y-intercept when utilization rate is 0
+     */
+    uint256 public baseRatePerBlockForStable;
+
+    /**
+     * @notice The multiplier of utilization rate that gives the slope of the interest rate
+     */
+    uint256 public multiplierPerBlock;
+
+    /**
+     * @notice The multiplier of utilization rate that gives the slope of the stable interest rate
+     */
+    uint256 public multiplierPerBlockForStable;
+
+    /**
+     * @notice The multiplierPerBlock after hitting a specified utilization point for stable rate
+     */
+    uint256 public jumpMultiplierPerBlockForStable;
+
+    /**
+     * @notice The utilization point at which the jump multiplier is applied for stable rate
+     */
+    uint256 public kink;
+
+    /**
+     * @notice The premium rate applicable on optimal stable loan rate
+     */
+    uint256 public stableRatePremium;
+
+    /**
+     * @notice Optimal Stable Loan Rate
+     */
+    uint256 public optimalStableLoanRate;
+
+    /**
+     * @notice The address of the owner, i.e. the Timelock contract, which can update parameters directly
+     */
+    address public owner;
+
+    /**
+     * @param _baseRatePerBlockForStable The approximate target base APR, as a mantissa (scaled by BASE)
+     * @param _multiplierPerBlock The rate of increase in interest rate wrt utilization (scaled by BASE)
```

Comment:
should we write out with respect to. to save space we could also write per which is not an acronym